web/services/views.py

```diff
@@ -63,6 +64,9 @@ def get(self, request):
 
     def post(self, request):
         up_file = request.FILES['file']
+        if up_file.size > 1.049e+7:
```

Comment:
This looks like a bit of a blackbox number. Could you give it a name and put it has an upper-cased, module-level variable, e.g. `RECOVERY_FILE_SIZE_MAX_BYTES = 10*1024*1024`?